src/theming/sap_fiori_3_dark.scss

```diff
@@ -102,4 +102,7 @@
   /* Calendar */
   --fdCalendar_Text_Shadow: none;
   --fdCalendar_Active_Item_Border: none;
+
+  /* Grid List */
+  --fdGrid_List_SingleSelect_Hover_BoxShadow: 0 0.125rem 0.5rem 0 fade(var(--sapContent_ShadowColor), 30);
```

Comment:
`fade()` is a less function right? If it is, let's get rid of it - we can make a new variable with the output of `fade(var(--sapContent_ShadowColor), 30);`. The designers are supposed to be removing all less functions from the design specs.

Reply:
+1 
You can contact @jessmetivier and she will handle the communication with the designers.

Reply:
Asked @jessmetivier to help
thanks for the comment :slightly_smiling_face: